pkg/cvo/status.go

```diff
@@ -91,6 +92,7 @@ func mergeOperatorHistory(config *configv1.ClusterVersion, desired configv1.Rele
 				last.CompletionTime = &now
 			}
 		}
+		last.AcceptedRisks = acceptedRisksMsg
```

Comment:
I'm not sure it's actually this line or not, but in [the upgrade-out-of-change run][1], we're getting the target-version risk applied to both the initial and target version history entries:

```console
$ curl -s https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_cluster-version-operator/841/pull-ci-openshift-cluster-version-operator-master-e2e-agnostic-upgrade-out-of-change/1575564385961644032/artifacts/e2e-agnostic-upgrade-out-of-change/gather-extra/artifacts/clusterversion.json | jq -r '.items[].status.history[] | .version + " " + .image + "\n" + .acceptedRisks'
4.12.0-0.ci.test-2022-09-29-191341-ci-op-3g21i4r7-initial registry.build04.ci.openshift.org/ci-op-3g21i4r7/release@sha256:4f03d53b58580bad4ecdd4812ceb3367d5d82d7005265fa62400fb89789685dd

Target release version="" image="registry.build04.ci.openshift.org/ci-op-3g21i4r7/release@sha256:4f03d53b58580bad4ecdd4812ceb3367d5d82d7005265fa62400fb89789685dd" cannot be verified, but continuing anyway because the update was forced: unable to verify sha256:4f03d53b58580bad4ecdd4812ceb3367d5d82d7005265fa62400fb89789685dd against keyrings: verifier-public-key-redhat
[2022-09-29T20:33:08Z: prefix sha256-4f03d53b58580bad4ecdd4812ceb3367d5d82d7005265fa62400fb89789685dd in config map signatures-managed: no more signatures to check, 2022-09-29T20:33:08Z: unable to retrieve signature from https://storage.googleapis.com/openshift-release/official/signatures/openshift/release/sha256=4f03d53b58580bad4ecdd4812ceb3367d5d82d7005265fa62400fb89789685dd/signature-1: no more signatures to check, 2022-09-29T20:33:08Z: unable to retrieve signature from https://mirror.openshift.com/pub/openshift-v4/signatures/openshift/release/sha256=4f03d53b58580bad4ecdd4812ceb3367d5d82d7005265fa62400fb89789685dd/signature-1: no more signatures to check, 2022-09-29T20:33:08Z: parallel signature store wrapping containers/image signature store under https://storage.googleapis.com/openshift-release/official/signatures/openshift/release, containers/image signature store under https://mirror.openshift.com/pub/openshift-v4/signatures/openshift/release: no more signatures to check, 2022-09-29T20:33:08Z: serial signature store wrapping config maps in openshift-config-managed with label "release.openshift.io/verification-signatures", parallel signature store wrapping containers/image signature store under https://storage.googleapis.com/openshift-release/official/signatures/openshift/release, containers/image signature store under https://mirror.openshift.com/pub/openshift-v4/signatures/openshift/release: no more signatures to check]
Forced through blocking failures: Precondition "ClusterVersionRecommendedUpdate" failed because of "NoChannel": Configured channel is unset, so the recommended status of updating from 4.12.0-0.ci.test-2022-09-29-192530-ci-op-3g21i4r7-latest to 4.12.0-0.ci.test-2022-09-29-191341-ci-op-3g21i4r7-initial is unknown.
4.12.0-0.ci.test-2022-09-29-192530-ci-op-3g21i4r7-latest registry.build04.ci.openshift.org/ci-op-3g21i4r7/release@sha256:22aa2f6116cee104ab57bd5a4b36f52a807417a2c337643c3a68012499d4b39c

Target release version="" image="registry.build04.ci.openshift.org/ci-op-3g21i4r7/release@sha256:4f03d53b58580bad4ecdd4812ceb3367d5d82d7005265fa62400fb89789685dd" cannot be verified, but continuing anyway because the update was forced: unable to verify sha256:4f03d53b58580bad4ecdd4812ceb3367d5d82d7005265fa62400fb89789685dd against keyrings: verifier-public-key-redhat
[2022-09-29T20:33:08Z: prefix sha256-4f03d53b58580bad4ecdd4812ceb3367d5d82d7005265fa62400fb89789685dd in config map signatures-managed: no more signatures to check, 2022-09-29T20:33:08Z: unable to retrieve signature from https://storage.googleapis.com/openshift-release/official/signatures/openshift/release/sha256=4f03d53b58580bad4ecdd4812ceb3367d5d82d7005265fa62400fb89789685dd/signature-1: no more signatures to check, 2022-09-29T20:33:08Z: unable to retrieve signature from https://mirror.openshift.com/pub/openshift-v4/signatures/openshift/release/sha256=4f03d53b58580bad4ecdd4812ceb3367d5d82d7005265fa62400fb89789685dd/signature-1: no more signatures to check, 2022-09-29T20:33:08Z: parallel signature store wrapping containers/image signature store under https://storage.googleapis.com/openshift-release/official/signatures/openshift/release, containers/image signature store under https://mirror.openshift.com/pub/openshift-v4/signatures/openshift/release: no more signatures to check, 2022-09-29T20:33:08Z: serial signature store wrapping config maps in openshift-config-managed with label "release.openshift.io/verification-signatures", parallel signature store wrapping containers/image signature store under https://storage.googleapis.com/openshift-release/official/signatures/openshift/release, containers/image signature store under https://mirror.openshift.com/pub/openshift-v4/signatures/openshift/release: no more signatures to check]
Forced through blocking failures: Precondition "ClusterVersionRecommendedUpdate" failed because of "NoChannel": Configured channel is unset, so the recommended status of updating from 4.12.0-0.ci.test-2022-09-29-192530-ci-op-3g21i4r7-latest to 4.12.0-0.ci.test-2022-09-29-191341-ci-op-3g21i4r7-initial is unknown.
```

[1]: https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_cluster-version-operator/841/pull-ci-openshift-cluster-version-operator-master-e2e-agnostic-upgrade-out-of-change/1575564385961644032